reactioncommerce/reaction
# chore: update js-doc throughout the app, batch 1

Resolves parts of #5269 & #5267   
Impact: **major**  
Type: **docs|chore**

## Issue
We are missing entire `jsdoc` blocks in some instances, and param / return / description lines of `jsdoc` in other instances, along with other `jsdoc` errors, all throughout Reaction. This is the biggest contributor to our `eslint` warnings.

## Solution
Add / fix `jsdoc` where necessary.
Since there are so many of these warnings, I'm going to split it up into multiple PR's to keep the file count low for easier viewing / less conflicts with open branches.

## Breaking changes
None

## Testing
1. Start the app
1. Make sure it runs
1. All changes are documentation changes, there should be no effect on the app in general

More detail for what each of these sections should include are available in our [Contributing Docs](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction)
